.golangci.yml

```diff
@@ -0,0 +1,95 @@
+run:
+  tests: true
+  timeout: 5m
+  skip-files:
+    - ".*\\.pb\\.go"
+  skip-dirs-use-default: true
+
+linters-settings:
+  goimports:
+    local-prefixes: github.com/antoninbas/p4runtime-go-client/
+
+linters:
+  disable-all: true
+  enable: # see https://golangci-lint.run/usage/linters/
+    #
```

Comment:
I don't think I see a reason to provide the full list of all the linters commented out. I think we should just keep the ones we use. You already have the link above (`https://golangci-lint.run/usage/linters/`) if someone needs to enable a new one.

Reply:
Ack!

Reply:
> Ack!

LOL. Your same replies got me laughing in the middle of the night